### PR TITLE
Remove PSDefaultParameterValues clear on import.

### PR DESCRIPTION
**Pull request by swernli**
@jterry75 
We don't need that line per @lzybkr

### CODE REVIEW

File: src/Docker.PowerShell/Docker.psm1 [hunk 1]
```diff
@@ -6,6 +6,7 @@
 # the module is in.
 #
 
+$paramBackup = $PSDefaultParameterValues.Clone()
```
Comment (lzybkr) [inline src/Docker.PowerShell/Docker.psm1]: Are you trying to protect your code from a real customer reported issue or a theoretical issue?

Most modules don't bother doing anything with $PSDefaultParameterValues, and this module is very simple, very few conflicts to worry about.

Comment (swernli) [inline src/Docker.PowerShell/Docker.psm1]: It's in the published guidance from the powershell team:
https://msdn.microsoft.com/en-us/powershell/gallery/psget/module/modulewithpseditionsupport#option-1-packaging-a-module-for-targeting-multiple-versions-and-multiple-editions-of-powershell

Comment (lzybkr) [inline src/Docker.PowerShell/Docker.psm1]: Actually, each module gets its own copy of `PSDefaultParameterValues`, so clearing is a nop. You can just delete the line of code and not worry about it.

Comment (swernli) [inline src/Docker.PowerShell/Docker.psm1]: Works for me.

**Comment by jterry75** [comment]
LGTM

**Pull request merged**